### PR TITLE
Link funds screen UI

### PR DESCRIPTION
**Pull request by manhduc-nguyen**
Screen Shot 2022-01-17 at 22 02 02

will optimize some common components in next PR

### CODE REVIEW

File: lib/app/cubit/link_funds/link_your_bank_cubit.dart [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import 'package:beanstalk_mobile/app/cubit/link_funds/link_your_bank_state.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LinkFundsCubit extends Cubit<LinkFundsState> {
```
Comment (mduccc) [inline lib/app/cubit/link_funds/link_your_bank_cubit.dart:4]: `Class name` and `file name` are conflict:
`link_your_bank_cubit.dart` should be `link_funds_cubit.dart`

File: lib/app/cubit/link_funds/link_your_bank_state.dart [hunk 2]
```diff
@@ -0,0 +1,3 @@
+abstract class LinkFundsState {}
+
+class LinkFundsInitState extends LinkFundsState {}
```
Comment (mduccc) [inline lib/app/cubit/link_funds/link_your_bank_state.dart:3]: Same!
`link_your_bank_state.dart` should be `link_funds_state.dart`

File: lib/app/page/link_funds/link_funds_page.dart [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                  'Link Funds Instantly',
+                  textAlign: TextAlign.left,
+                  style: TextStyle(
+                    color: DSColors.stone,
+                    fontSize: 24,
+                    fontWeight: FontWeight.w500,
+                  ),
+                ),
+              ),
+              Container(
```
Comment (mduccc) [inline lib/app/page/link_funds/link_funds_page.dart:46]: Consider separate the widget. Then can reuse.

Comment (manhduc-nguyen) [reply lib/app/page/link_funds/link_funds_page.dart:46]: yes I will optimize common components in next PR like I said

File: lib/app/cubit/link_funds/link_your_bank_cubit.dart (hunk 1 above)
Comment (manhduc-nguyen) [reply lib/app/cubit/link_funds/link_your_bank_cubit.dart:4]: nice, forgot to change it.

**Pull request merged**